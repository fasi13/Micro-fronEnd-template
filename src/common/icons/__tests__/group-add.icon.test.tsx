import { render } from '@testing-library/react';
import React from 'react';
import { GroupAddIcon } from '../index';

test('It should render GroupAddIcon', async () => {
	const size = { width: 512, height: 512 };

	expect(
		render(
			<GroupAddIcon className="" width={size.width} height={size.height} />,
		),
	).toMatchSnapshot();
});
