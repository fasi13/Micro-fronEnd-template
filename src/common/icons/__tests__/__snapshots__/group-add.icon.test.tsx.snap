// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`It should render GroupAddIcon 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <svg
        class=""
        fill="none"
        height="512"
        viewBox="0 0 112 112"
        width="512"
        xmlns="http://www.w3.org/2000/svg"
      >
        <g
          clip-path="url(#clip0)"
        >
          <path
            d="M109.154 29.881L57.949 7.047a4.766 4.766 0 00-3.902 0L2.844 29.88a4.792 4.792 0 00-.302 8.608l51.202 27.306a4.76 4.76 0 002.256.564c.78 0 1.552-.185 2.26-.564l51.204-27.306a4.8 4.8 0 002.535-4.395 4.81 4.81 0 00-2.845-4.213z"
            fill="#fff"
          />
          <path
            d="M104.945 50.313l-48.95 26.112L7.053 50.313a4.786 4.786 0 00-6.486 1.975 4.788 4.788 0 001.973 6.485l51.202 27.314a4.79 4.79 0 002.256.564c.78 0 1.553-.185 2.26-.564l51.204-27.314a4.796 4.796 0 10-4.517-8.46z"
            fill="#fff"
          />
          <path
            d="M104.945 69.042l-48.95 26.104L7.053 69.042a4.787 4.787 0 00-6.486 1.972A4.789 4.789 0 002.54 77.5l51.202 27.305c.706.379 1.48.567 2.256.567.78 0 1.553-.185 2.26-.567L109.462 77.5a4.8 4.8 0 001.974-6.486 4.795 4.795 0 00-6.491-1.972z"
            fill="#fff"
          />
          <path
            d="M76.152 31.033H35.848v9.939h40.304v-9.939z"
            fill="#3F98E3"
          />
          <path
            d="M60.972 15.848h-9.939v40.304h9.939V15.848z"
            fill="#3F98E3"
          />
        </g>
        <defs>
          <clippath
            id="clip0"
          >
            <path
              d="M0 0h112v112H0z"
              fill="#fff"
            />
          </clippath>
        </defs>
      </svg>
    </div>
  </body>,
  "container": <div>
    <svg
      class=""
      fill="none"
      height="512"
      viewBox="0 0 112 112"
      width="512"
      xmlns="http://www.w3.org/2000/svg"
    >
      <g
        clip-path="url(#clip0)"
      >
        <path
          d="M109.154 29.881L57.949 7.047a4.766 4.766 0 00-3.902 0L2.844 29.88a4.792 4.792 0 00-.302 8.608l51.202 27.306a4.76 4.76 0 002.256.564c.78 0 1.552-.185 2.26-.564l51.204-27.306a4.8 4.8 0 002.535-4.395 4.81 4.81 0 00-2.845-4.213z"
          fill="#fff"
        />
        <path
          d="M104.945 50.313l-48.95 26.112L7.053 50.313a4.786 4.786 0 00-6.486 1.975 4.788 4.788 0 001.973 6.485l51.202 27.314a4.79 4.79 0 002.256.564c.78 0 1.553-.185 2.26-.564l51.204-27.314a4.796 4.796 0 10-4.517-8.46z"
          fill="#fff"
        />
        <path
          d="M104.945 69.042l-48.95 26.104L7.053 69.042a4.787 4.787 0 00-6.486 1.972A4.789 4.789 0 002.54 77.5l51.202 27.305c.706.379 1.48.567 2.256.567.78 0 1.553-.185 2.26-.567L109.462 77.5a4.8 4.8 0 001.974-6.486 4.795 4.795 0 00-6.491-1.972z"
          fill="#fff"
        />
        <path
          d="M76.152 31.033H35.848v9.939h40.304v-9.939z"
          fill="#3F98E3"
        />
        <path
          d="M60.972 15.848h-9.939v40.304h9.939V15.848z"
          fill="#3F98E3"
        />
      </g>
      <defs>
        <clippath
          id="clip0"
        >
          <path
            d="M0 0h112v112H0z"
            fill="#fff"
          />
        </clippath>
      </defs>
    </svg>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
